packages/payload/src/admin/components/forms/field-types/Relationship/index.tsx

```diff
@@ -139,7 +140,14 @@ const Relationship: React.FC<Props> = (props) => {
 
           if (resultsFetched < 10) {
             const collection = collections.find((coll) => coll.slug === relation)
-            const fieldToSearch = collection?.admin?.useAsTitle || 'id'
+            let fieldToSearch
+            if (typeof sortOptions === 'string') {
+              fieldToSearch = sortOptions
+            } else if (sortOptions && typeof sortOptions === 'object') {
+              fieldToSearch = sortOptions[relation] || collection?.admin?.useAsTitle || 'id'
+            } else {
+              fieldToSearch = collection?.admin?.useAsTitle || 'id'
```

Comment:
I'm wondering if there is no `admin.sortOptions` property defined, shouldn't it be up the to collection `defaultSort` to be used instead? If so then the backend would do that for you without all the `||` situatoins.

Reply:
@DanRibbens Prior to this change, relationship drop downs were defaulting to either `useAsTitle` || `'id'` even if a `defaultSort` was specified by the related collection. This new property gives users the option to sort the drop downs however they please e.g. matching it to the defaultSort if the collection is using one.

Reply:
From a code perspective this could be a bit neater:

```ts
let fieldToSearch = fieldToSearch = collection?.admin?.useAsTitle || 'id'
if (typeof sortOptions === 'string') {
  fieldToSearch = sortOptions
} else if (sortOptions?.[relation]) {
  fieldToSearch = sortOptions[relation]
}
```

But I think Dan brings up a good point, there is a default sort property on collections. Should we be using that as one of the `||` statements when we initially set `fieldToSearch`?

Here is a link to the docs section about defaultSort: https://payloadcms.com/docs/configuration/collections#options

Reply:
@DanRibbens @JarrodMFlesch I went ahead and added the above suggested changes. Thank you.